stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -136,6 +149,12 @@ swift_install_symlink_component(clang-resource-dir-symlink
   TARGET ../clang/${CLANG_VERSION}
   DESTINATION "lib/swift")
 
+# Adding link to "llvm" at install time.
+swift_install_symlink_component(llvm-resource-dir-symlink
+  LINK_NAME llvm
+  TARGET ../
+  DESTINATION "lib/swift")
```

Comment:
This just points to `lib/`. Just to check, is that where libFuzzer goes in an LLVM install?

(I'm not convinced it makes sense to call this an "LLVM resource directory" in that case. "resource dir" actually means something for Clang because (like Swift) Clang has a `-resource-dir` flag.)